app/javascript/app/components/nav/nav.js

```diff
@@ -1,12 +1,44 @@
+import { createElement, PureComponent } from 'react';
 import { connect } from 'react-redux';
-
+import PropTypes from 'prop-types';
+import { setUser, navigateToLogin } from '../../pages/login/login-actions';
+import { logout } from '../../services/login.service';
```

Comment:
could you change relative imports for absolute? 
as we already have the aliases it will be more readable and maintainable